domain/usecases/account/accounts.go

```diff
@@ -0,0 +1,76 @@
+package account
+
+import (
+	"errors"
+
+	"github.com/Vivi3008/apiTestGolang/domain/entities/account"
+)
+
+type AccountUsecase struct {
+	repo account.AccountRepository
+}
+
+var (
+	ErrInsufficientLimit = errors.New("insufficient Limit")
+	ErrValueEmpty        = errors.New("value is empty")
+)
+
+type MethodPayment string
+
+const (
+	Debit  MethodPayment = "Débito"
+	Credit MethodPayment = "Crédito"
+)
+
+func CreateNewAccountUsecase(acc account.AccountRepository) AccountUsecase {
```

Comment:
O que acha de renomear esse construtor aqui?
```suggestion
func NewAccountUsecase(acc account.AccountRepository) AccountUsecase {
```

Reply:
Boa ideia! 👍🏽